app/utils/eventStatus.js

```diff
@@ -23,14 +22,36 @@ const eventStatus = (event: Event) => {
     case 'OPEN':
       return 'Åpent arrangement';
     case 'INFINITE':
+      if (activationTime === null) {
+        return 'Ingen påmeldingsrett';
+      }
       return 'Åpent med påmelding';
     case 'NORMAL':
+      if (activationTime === null) {
```

Comment:
What will happen if the user has registered, and `activationTime` is (then) null?

Reply:
Added logic in 6ff1375.